website/content/en/docs/operator-scope.md

```diff
@@ -6,7 +6,7 @@ weight: 50
 
 - [Namespace-scoped operator usage](#namespace-scoped-operator-usage)
 - [Cluster-scoped operator usage](#cluster-scoped-operator-usage)
-  - [Changed required for a cluster-scoped operator](#changed-required-for-a-cluster-scoped-operator)
+  - [Changes required for a cluster-scoped operator](#changes-required-for-a-cluster-scoped-operator)
   - [Example for cluster-scoped operator](#example-for-cluster-scoped-operator)
 - [CRD scope](#crd-scope)
   - [CRD cluster-scoped usage](#crd-cluster-scoped-usage)
```

Comment:
the index here we can remove. Since all the other docs have not and then, IHMO we should keep the same standard across the project docs

Reply:
I think you're right. Ill do that in a follow-up PR.